Makefile

```diff
@@ -104,3 +105,11 @@ check: test lint
 format:
 	(cd provider && go fmt ./...)
 	shfmt -w .githooks/*
+
```

Comment:
Let's try to keep our formatting consistent.  One newline is enough between the previous and next target.